keel/eligibility_calculator/models.py

```diff
@@ -1,13 +1,16 @@
 from django.db import models
+from django.utils import timezone
 from keel.leads.models import CustomerLead
-
+from keel.authentication.models import TimeStampedModel
 # Create your models here.
 
 
 class EligibilityResults(models.Model):
 
     lead_id = models.ForeignKey(CustomerLead, on_delete=models.DO_NOTHING, related_name="lead_id", null=True)
     data = models.JSONField(verbose_name="Data", null=True)
+    created_at = models.DateTimeField(default=timezone.now)
+    updated_at = models.DateTimeField(default=timezone.now)
```

Comment:
if you are using the TimeStampedModel , you just need to inherit the class from that model
not create the fields manually

Reply:
done